packages/permission-controller/src/PermissionController.ts

```diff
@@ -2185,10 +2186,12 @@ export class PermissionController<
     }
 
     try {
-      this.messagingSystem.call(
```

Comment:
Is the `await` intentionally being omitted here? This seems like a bug

Reply:
I didn't want to mess with this in case there was a specific reason, but it did seem odd to me, too.